src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -170,6 +172,31 @@ const SpendingByCFDAContainer = ({ activeTab, scrollIntoView }) => {
         });
     };
 
+    const addUnlinkedData = (rows, totals) => {
+        // add unlinked data if activeTab is all
+        const table = document.getElementsByClassName('assistance-listing')[0];
+
+        if (activeTab === 'all') {
+            const unlinkedData = calculateUnlinkedTotals(assistanceTotals, totals);
+            table.classList.add('unlinked-data');
```

Comment:
Here, we shouldn't be interacting with the dom. We can add a ternary to the table inside the `className` attribute. This is an anti pattern.